src/main/java/ch/njol/skript/conditions/CondHasBuildRestrictions.java

```diff
@@ -0,0 +1,43 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.NotNull;
+
+@Name("Has Build Restrictions")
+@Description("Check if an item has any build restrictions.")
```

Comment:
It seems like it would be useful to check specific build restrictions as well, e.g. `if ... can be placed on stone` or `... can break oak log`.

Reply:
Would this be best implemented by switching from a property expression so I can register multiple patterns and then handle each one? Or is there some better way to approach this?